### ISSUE TITLE
get_bucket_policy() doesn't return correct output in one case

### ISSUE THREAD

**Issue by vadmeste** [minio/minio-py#630]
The following code shows a bug in  get_bucket_policy()  after setting two bucket policies with different prefixes. 

```
from minio import Minio
from minio.policy import Policy
from minio.error import ResponseError

client = Minio('s3.amazonaws.com', secure=True, access_key='xxx', secret_key='xxx/xxx/xxx')
bucket_name = 'mybucket'

try:
    client.set_bucket_policy(bucket_name, '', Policy.WRITE_ONLY)
    client.set_bucket_policy(bucket_name, 'public/', Policy.READ_ONLY)
    print(client.get_bucket_policy(bucket_name))
    print(client.get_bucket_policy(bucket_name, 'public/'))
except ResponseError as err:
    print(err)
```

The code above returns the following output:
```
none
readonly
```
which is wrong, it should be:

```
writeonly
readonly
```

**Comment by vadmeste** [comment]
Looks like we don't need this anymore after bucket policy API simplification.

**Comment by ebozduman** [comment]
That is correct. 
I started working on issue #635. 
This one can be closed as duplicate of #635.
